build-system/tasks/performance/config.json

```diff
@@ -8,7 +8,10 @@
     },
     {
       "handlerName": "analyticsHandler",
-      "urls": [],
+      "urls": [
+        "http://localhost:8000/test/manual/amp-analytics/performanceTestPages/performance-longer.html",
```

Comment:
visible-longer?

Reply:
ugh last minute name changes... thanks for the catch

Reply:
yeah it happens, a round of sanity check would be helpful.